frontend/src/component/insights/Insights.tsx

```diff
@@ -32,10 +34,20 @@ export const Insights: FC<InsightsProps> = ({ withCharts = true }) => {
 
     const stateConfig = {
         project: FilterItemParam,
-        from: FilterItemParam,
-        to: FilterItemParam,
+        from: withDefault(FilterItemParam, {
+            values: [format(subMonths(new Date(), 1), 'yyyy-MM-dd')],
+            operator: 'IS',
+        }),
+        to: withDefault(FilterItemParam, {
+            values: [format(new Date(), 'yyyy-MM-dd')],
+            operator: 'IS',
+        }),
     };
-    const [state, setState] = usePersistentTableState('insights', stateConfig);
+    const [state, setState] = usePersistentTableState('insights', stateConfig, [
+        'from',
```

Comment:
we don't want o persist from and to to local storage and prefer to always use the moving last month window